### PR TITLE
DEVCVL-263 - Default network BSC mainnet when in PROD or UAT | Loading issue resolved

### PR DESCRIPTION
**Pull request by shakil-adv**

### CODE REVIEW

**Comment by kyawthetkt** [comment]
image
If logged in, two drop down of chain appears.

File: src/components/header/Header.js [hunk 1]
```diff
@@ -18,6 +18,13 @@ const Header = () => {
     const { isAuthenticated, isWeb3Enabled, isWeb3EnableLoading } =
         useMoralis();
 
+    if (
+        window.location.href === "https://marketplace.canvasland.io/" ||
+        window.location.href === "http://marketplace1.canvasland.io/"
+    ) {
+        switchNetwork("0x38");
```
Comment (kyawthetkt) [inline src/components/header/Header.js:25]: `window.location.href === "https://marketplace.canvasland.io/" ||
        window.location.href === "http://marketplace1.canvasland.io/"`

For `http://marketplace1.canvasland.io/`,it I think should be BSC Testnet.

File: src/components/header/Header.js [hunk 2]
```diff
@@ -26,8 +26,6 @@ const Header = () => {
     }
 
     const onClick = ({ key }) => {
-        console.log(key);
-        await enableWeb3();
         switchNetwork(key);
     };
```
[the remark this answers is not in the document]
Comment (shakil-adv) [reply src/components/header/Header.js:29]: I've removed that, somehow it is back after the manual merge. Removed again

**Comment by shakil-adv** [comment]
> image If logged in, two drop down of chain appears.

I've removed that, somehow it is back after the manual merge. Removed again

**Review by kyawthetkt — APPROVED** [review]

File: src/components/header/Header.js (hunk 1 above)
Comment (jibrail-idris) [reply src/components/header/Header.js:22]: Simple and straightforward.

**Review by pracheebapate — APPROVED** [review]

**Review by jibrail-idris — APPROVED** [review]

**Pull request merged**